brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-3336 Fix: Image gallery - Resolved an issue of Image gallery lightbox overlay color is not displaying properly, it hide the page content.

### Description

- https://brainstormforce.slack.com/archives/C03TMLQG0EN/p1694338763249929

### Screenshots

- https://d.pr/v/VtqzXn

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop container
- Use Image gallery block with lightbox enable.
- Visit frontend, Click on any image from Image gallery.
- Make sure content behind the Lightbox visible, should not hidden.
- Please check screencast for more explanation

### Checklist:
